tests/e2e/tracer/infrastructure.py

```diff
@@ -0,0 +1,17 @@
+from pathlib import Path
+
+from tests.e2e.utils.data_builder import build_service_name
+from tests.e2e.utils.infrastructure import BaseInfrastructureV2
+
+
+class TracerStack(BaseInfrastructureV2):
+    # Maintenance: Tracer doesn't support dynamic service injection (tracer.py L310)
+    # we could move after handler response or adopt env vars usage in e2e tests
+    SERVICE_NAME: str = build_service_name()
+
+    def __init__(self, handlers_dir: Path, feature_name: str = "tracer") -> None:
+        super().__init__(feature_name, handlers_dir)
+
+    def create_resources(self):
```

Comment:
Should we add  -> None to notify reader that this function doesnt return anything?

Reply:
Done. `None` is implicit but it doesn't hurt being explicit that they don't have to return anything. Good catch!